src/sentry/testutils/fixtures.py

```diff
@@ -700,3 +702,24 @@ def add_user_permission(self, user, permission):
                 UserPermission.objects.create(user=user, permission=permission)
         except IntegrityError:
             raise
+
+    def create_sentry_app(self, name=None, organization=None, published=False, scopes=(),
+                          webhook_url=None, **kwargs):
+        if not name:
+            name = 'Test App'
```

Comment:
can you just set 'Test App' and 'example.com' as default values? (not org though, because it is evaluated at import time and self.organization may not be valid yet)

Reply:
wait why wouldn't self.organization be valid yet if it's using https://github.com/getsentry/sentry/blob/master/src/sentry/testutils/fixtures.py#L180 ?